docs/yml.md

```diff
@@ -87,6 +87,23 @@ external_links:
  - project_db_1:postgresql
 ```
 
+### extra_hosts
+
+Add hostname mappings. Use the same values as the docker client `--add-host` parameter.
+
+```
+extra_hosts:
+ - "docker:162.242.195.82"
+ - "fig:50.31.209.229"
+```
```

Comment:
"docker" and "fig" are strange host names - why would I have a host called "fig"? (cc @moxiegirl - any best practices here?)

Could you add an example with a dict? Best if the strings are quoted, e.g.

```
extra_hosts:
  somehost: "162.242.195.82"
  otherhost: "50.31.209.229"
```